content/learn/beginner/outputs-1.md

```diff
@@ -0,0 +1,85 @@
++++
+title = "Outputs 1"
+weight = 210
++++
+
+In this chapter, we will go over two ways to output text from a custom command: using the *response*, and later on using
+*template actions*.
+
+## Response
+
+If you paste text into the response field of a custom command, the bot will respond with that text when the command is
+triggered.
+
+For instance, paste the following text into a new custom command:
+
+```go
+Hello World.
+```
+
+This will make the bot respond "Hello World." in a new message when the command is triggered, as shown below.
+
+<center>
+
+![Hello World example ran in Discord.](hello_world.png)
+
+</center>
+
+## Template Actions
+
+### Actions As Output
+
+Earlier, we just made the bot respond with some static text. However, sometimes (or rather, most of the time) we want to
+change that around depending on various factors. This is where *template actions* come in.
+
+Template actions are a way to dynamically change the output depending on various things, such as the user who triggered
+the command, the arguments passed to the command, or even the current time.
```

Comment:
"depending on environmental data"

I don't particularly like the word "environmental", but something better than "various things" is needed.